pages/common/luac.md

```diff
@@ -0,0 +1,11 @@
+# luac
+
+> Lua byte code compiler.
+
+- Compile lua source code to lua byte code:
+
+`luac -o {{byte_code.luac}} {{source.lua}}`
+
+- Stop including debugging information in byte code:
```

Comment:
It probably reads better if you say "Strip debug information from the output" or "Don't include debug information in the byte code".  "Stop including.." implies some sort of a lasting effect..

Reply:
I'd go for "Don't include debug information in the byte code"